content/algorithm-and-data-structures/index.mdx

```diff
@@ -1,8 +1,56 @@
 ---
-title: Algorithm & Data-Structures
-tags: []
+title: Algorithm and data structures
+tags:
+  [
+	Sorting Algorithms,
+	Linked Lists,
+	Trees,
+	Landau-Symbols,
+	Pathfinding,
+	Edit-Distance,
+	Hashing Algorithms
+  ]
 chapter: 8
 icon: ⛓
 ---
 
-## coming soon...
+Computer Graphics is often seen as one of the hardest subjects to learn. This page will focus on OpenGL which is the library that is also used in the courses.
+This page gives you a brief overiew of the most important topics in Algortihm and Datastrucure Basics. Links for further research will also be included
+
+## Sorting Algorithms
+
+Depending on the underlying Data, there are various Algorithms that can be used to arrange them in an ordered way, going from Basic 1 by 1 Sorting to advanced complex procedures.
```

Comment:
A nice addition would be to list all of the different sorting algorithms. 
Maybe with some pseudo code links too.